src/platform/user/profile/constants/addressValidationMessages.js

```diff
@@ -0,0 +1,30 @@
+export const addressValidationMessages = {
```

Comment:
I love me some strings stored in constants. I would suggest make the constant name SCREAMING_SNAKE_CASE, per our unwritten convention. And maybe freeze the object. ex: https://github.com/department-of-veterans-affairs/vets-website/blob/84da9c4d377f665a5943be76f32964d5b71f9157/src/applications/hca/constants.js#L1

Reply:
Ideally the keys themselves would also be stored in constants as well since those are being used elsewhere. I remember doing that on a smaller scale here https://github.com/department-of-veterans-affairs/vets-website/blob/8a80ca83181dd3e3b88d327904b1503c046bb21f/src/applications/personalization/dashboard/components/ESRError.jsx#L4